documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -0,0 +1,1946 @@
+[[ogm-redis]]
+
+== Redis (Experimental)
+
+http://redis.io/[Redis] is a key-value datastore
+which stores your data in a variety of data structures. Although there is no one-and-only style
+how to map data between data structures and Redis, two major styles are are most common:
+
+* Storing entities as JSON documents
+* Storing entities as a bunch of String-based key-value pairs
+
+[NOTE]
+====
+Support for Redis is considered an EXPERIMENTAL feature as of this release.
+In particular you should be prepared for possible changes to the persistent representation of mapped objects in future releases.
+
+Also be aware of the fact that partial updates are unsupported at the moment
```

Comment:
Yeah, but I didn't take removals/set to null operations into account yet. My bad.